.gitignore

```diff
@@ -16,6 +16,7 @@
 npm-debug.log*
 yarn-error.log
 testem.log
+.vscode/
```

Comment:
This stuff should be move to your global `.gitignore` file (see https://help.github.com/articles/ignoring-files/)

Only project specific files should be added to this file, but no individual editor config files etc.